soccer/planning/tests/PlannerTest.cpp

```diff
@@ -267,16 +277,17 @@ TEST(Planning, settle_basic) {
                         nullptr};
     SettlePlanner planner;
     Trajectory path = planner.plan(std::move(request));
-    EXPECT_TRUE(checkPathContinuous(path, RobotConstraints{}));
+    EXPECT_TRUE(checkTrajectoryContinuous(path, RobotConstraints{}));
 }
 
 TEST(Planning, settle_pointless_obs) {
     WorldState world_state;
-    world_state.ball.position = Point{1, 1};
-    world_state.ball.velocity = Point{-.1, -.1};
+    // Use some initial velocity, settle doesn't always work for non-moving balls
```

Comment:
Make an issue?

Reply:
This is fine. Collect is intended to handle those cases.